README.md

```diff
@@ -8,16 +8,24 @@
 [![Crates.io](https://img.shields.io/crates/v/slide)](https://crates.io/crates/slide)
 [![Github help-wanted](https://img.shields.io/github/issues/yslide/slide/help%20wanted)](https://github.com/yslide/slide/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22)
 
-slide is an static expression optimizer. Given an expression like
+slide is an expression rewrite system and validator. Given an expression like
 
 ```
 x(x + 2 * 3) / (x + 6)
 ```
 
-slide should be able emit the lowered expression `x`.
+slide should be able emit the simplified expression `x`.
 
-One of slide's design goals is compilation as a platform, where optimizations are configurable
-plugins.
+slide's design goals include
+
+- Simplification as a platform, where rewrite rules are user-configurable.
+    For example, you should be able to give to (or remove from!) slide a rule like `x^2 dx -> 2x`,
```

Comment:
NIT: Don't use you should be able to, I would rephrase as
```
For example, users are able to give to (or remove from!) slide a rule like ... 
```

Reply:
well they are not able to yet ):